pfcpiface/upf.go

```diff
@@ -127,13 +128,23 @@ func NewUPF(conf *Conf, fp fastPath) *upf {
 		nodeID:            nodeID,
 		fastPath:          fp,
 		dnn:               conf.CPIface.Dnn,
+		peers:             conf.CPIface.Peers,
 		reportNotifyChan:  make(chan uint64, 1024),
 		maxReqRetries:     maxReqRetriesDefault,
 		respTimeout:       respTimeoutDefault,
 		enableHBTimer:     conf.EnableHBTimer,
 		hbInterval:        hbIntervalDefault,
 	}
 
+	if len(conf.CPIface.Peers) > 0 {
+		u.peers = make([]string, len(conf.CPIface.Peers))
+		nc := copy(u.peers, conf.CPIface.Peers)
+
+		if nc == 0 {
+			log.Fatalln("Failed to parse cpiface peers")
```

Comment:
This `fatal` log here makes CP peers a mandatory parameter, which must be provided for PFCP Agent to start. Otherwise, PFCP Agent fails to start. I'm not convinced that it should be the default behavior. We should be able to use PFCP Agent **without** UPF->SMF Association support (as we do so far). Hence, I propose to log just warning here, e.g.:

```suggestion
			log.Warn("Failed to parse cpiface peers, PFCP Agent will not initiate connection to N4 peers.")
```

Reply:
Done